PluginsAndFeatures/azure-toolkit-for-intellij/resources/whatsnew/whatsnew.md

```diff
@@ -1,10 +1,34 @@
-<!-- Version: 3.37.0 -->
+<!-- Version: 3.38.0 -->
 # What's new in Azure Toolkit for IntelliJ
 
 > Scroll down to checkout our newly added support for **Azure Functions**, **Azure Spring Cloud** and
  **log streaming for Azure App Service**.
 
-## 3.37.0(Current Release)
+## 3.38.0(Current Release)
+We are proud to announce the support for [Azure Synapse](https://docs.microsoft.com/en-us/azure/synapse-analytics/spark/intellij-tool-synapse) in Azure toolkit! The following feature is waiting to be discovered.
```

Comment:
following feature is -> following features are ?

Reply:
Good catch!